### DISCUSSION TITLE
Feature : Reports of M:M related module

### DISCUSSION THREAD

**Discussion by souravkumar4797** [YetiForceCompany/YetiForceCRM#17211] [How to]
Team, 

I was wondering how to create a widget or list view or summarize M:M related modules. I have checked with workflow, Filters & Widgets. Did any one of you have any idea how to achieve it.

Use Case 1: Suppose I have recorded multiple campaign under that I have linked multiple leads. 
I want to get a report of list of unique leads contacted in 2 of the campaigns. 
If it would be a 1:M relation it can be achieved by just modifying the filter of the view you can add related records fields.

Use Case 2: Summary : I want to get a pivot table or summary of campaigns it took to nurture a lead. may be a simple count in the lead record view.

**Comment by rskrzypczak** [comment]
You just chose a problematic case, because normally you could use advanced data filtering conditions in the list of records.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/cdf073bf-775f-40cb-be90-67af85960fcb)

However, the campaign relationship is custom and unavailable by default.
It would be best to improve the possibility of filtering with this relationship with the campaign, but whether someone will do it and when is another story :)

I don't really see the possibility of achieving what you want without any programming work
